### ISSUE TITLE
Semantics

### ISSUE THREAD

**Issue by Errytagedesign** [mumbereh/my-portfolio#7]
Please wrap your image tag in a `<figure> </figure>` rather than a `<div> </div>` to follow semantic rules

**Comment by Tupai566** [comment]
Please kindly ensure that images have the alt image text 👍 . This is because alt attribute messages enable screen readers to read the information about on-page images for the benefit of a person with a complete lack of sight, visually impaired, or who is otherwise unable to view the images on the page.

**Comment by mumbereh** [comment]
@Errytagedesign@Tupai566 
Thank you for your technical comments. I will address them to make my app look more professional.
